### PR TITLE
Avoid nesting

### PR DESCRIPTION
**Pull request by seblemaguer**
It is a bit patchy, but it aims to solves #137

### CODE REVIEW

**Review by jeremy-compostella — COMMENTED** [review]
Could you make a nice commit message?

**Comment by seblemaguer** [comment]
hello, sure I hope it is better :)

File: org-msg.el [hunk 1]
```diff
@@ -1223,27 +1223,28 @@ MML tags."
...
-	      (dolist (rep '(("^>+ *" . "") ("___+" . "---")))
-		(save-excursion
-		  (while (re-search-forward (car rep) nil t)
-		    (replace-match (cdr rep)))))
-	      (org-escape-code-in-region (point) (point-max))))
-	  (when .signature
-	    (unless (eq .style 'top-posting)
-	      (goto-char (point-max)))
-	    (insert .signature))
+	  (when (not (search-forward org-msg-options nil t))
```
Comment (jeremy-compostella) [inline org-msg.el:1226]: (when (not expr) body) <=> (unless expr body)

**Review by jeremy-compostella — COMMENTED** [review]
Could you briefly explain what it fixes ? a reference to the #156 in the commit message would also be helpful.

I added a code suggestion.

**Comment by seblemaguer** [comment]
Updated with the code suggestion (thanks for it :))

I didn't quite understand this bit
> a reference to the https://github.com/jeremy-compostella/org-msg/pull/156 in the commit message would also be helpful.

I think you meant #137 so I modified it this way. If that is not what you meant, I can reword the commit

**Pull request merged**